src/backend/gl/Cargo.toml

```diff
@@ -17,29 +17,25 @@ build = "build.rs"
 name = "gfx_backend_gl"
 
 [features]
-#default = ["wgl", "surfman", "surfman-x11"]
-default = []
-wgl = ["winapi"]
-#surfman-x11 = ["surfman/sm-x11"]
+default = ["surfman"]
+surfman-x11 = ["surfman/sm-x11"]
 
 [dependencies]
 arrayvec = "0.5"
 bitflags = "1"
-log = { version = "0.4" }
-gfx-hal = { path = "../../hal", version = "0.6" }
+log = "0.4"
+hal = { path = "../../hal", version = "0.6", package = "gfx-hal" }
 auxil = { path = "../../auxil/auxil", version = "0.7", package = "gfx-auxil", features = ["spirv_cross"] }
 smallvec = "1.0"
-glow = "0.4"
+glow = "0.6.1"
 parking_lot = "0.11"
 spirv_cross = { version = "0.22", features = ["glsl"] }
 lazy_static = "1"
 raw-window-handle = "0.3"
-# disable this for now, the versions of glutin are jumping too fast
-#glutin = { version = "0.24", optional = true }
 
-#[target.'cfg(all(unix, not(target_os = "ios")))'.dependencies]
+[target.'cfg(all(unix, not(target_os = "ios")))'.dependencies]
```

Comment:
what happens on Web/WASM and iOS targets? How does `surfman` as default dependency gets processed?

Reply:
I don't actually know. I know the cfg aliases in `build.rs` gate the code using `surfman` off from those platforms already, so it shouldn't change the generated code even if it were included on those platforms. I'm not sure if `cargo` would still pull the dependency, though. Maybe it doesn't matter and we should just leave the check out.

Reply:
I'm mostly wondering if we should have surfman as an optional not enabled by default dependency. `wgpu` for example will have no issues selecting it specifically when building on linux/macos on non-wasm.

Reply:
Ah, Got it. I'll just make it non-optional without the feature gate then.